### PR TITLE
Fix path for check log

### PR DESCRIPTION
**Pull request by mf69**
## Pull Request Checklist

**Is this in reference to an existing issue?**
Yes Plugin copies the file to /var/cache folder and does its checking #15

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/check-log.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
 require 'fileutils'
 
 class CheckLog < Sensu::Plugin::Check::CLI
-  BASE_DIR = '/var/cache/check-log'.freeze
+  BASE_DIR = '/var/cache/sensu/check-log'.freeze
```
Comment (majormoses) [inline bin/check-log.rb:36]: Is there a reason to not make this an option and default to this? I tend to dislike configuration that is not exposed.

**Comment by majormoses** [comment]
I tried running this check and I feel more strongly about it being an option I had no choice but to create the dir it wanted and chowned it. This is not a good experience and the user should be able to specify any dir they want as the base dir.

**Review by majormoses — CHANGES_REQUESTED** [review]
Remove `BASE_DIR` constant and make it an an option with a sane default.

**Comment by majormoses** [comment]
@mf69 any chance of picking this back up?

**Comment by mf69** [comment]
I don't really grok what you are asking for.  Perhaps provide an example from another sensu-plugin that works the way you specified?

**Comment by majormoses** [comment]
Remove this line: `BASE_DIR = '/var/cache/check-log'.freeze` and make an option that can have your proposed path as a default and be overwritten at runtime to provide more flexibility like most of things in this check.

**Comment by majormoses** [comment]
When I have some time I will do this right now I am closing this because of lack of activity and is the wrong approach to solve this.

**Pull request closed without merging**